### PR TITLE
[Feat #18]: Added 404 page

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Added 404 page
Screenshot (199)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -23,6 +24,7 @@ function App() {
         <Route path="/signup" element={<Signup />} />
         <Route path="/profile" element={<Profile />} />
         <Route path="/mockman" element={<Mockman />} />
+        <Route path="*" element={<NotFound />} />
       </Routes>
```
Comment (aniketsingh13) [inline src/App.js:28]: suggestion, you can make seperate folder for route

**Review by aniketsingh13 — APPROVED** [review]
code looks clean and ui also look good

File: src/App.js (hunk 1 above)
Comment (SanketDhabarde) [reply src/App.js:28]: Yes, will try.

**Review by ckahub — COMMENTED** [review]
code has good structure and paranthesis use
code is readable and easily understandable

**Review by TheAmlanSaikia — APPROVED** [review]
Good code. Things I like are-
-Elegant UI
-Clean and Structured Code
-Proper Variable names and Folder structure.

**Pull request merged**